### PR TITLE
EGD-8170

### PR DESCRIPTION
**Pull request by pholat**
crashes fix

### CODE REVIEW

File: .gdbinit
Comment (mjan-dev) [inline .gdbinit:3]: why this is removed?

File: module-apps/apps-common/windows/OptionWindow.hpp
Comment (SP2FET) [inline module-apps/apps-common/windows/OptionWindow.hpp:37]:
```suggestion
        /// otherwise destruction order will cause call to non existing parent element
```

**Review by kkleczkowski — APPROVED** [review]

File: module-apps/apps-common/windows/OptionWindow.hpp
Comment (SP2FET) [inline module-apps/apps-common/windows/OptionWindow.hpp:39]: almost like back to the future :D

**Review by SP2FET — APPROVED** [review]

**Review by mjan-dev — APPROVED** [review]
Approved, but please wait for response in the jira issue if there are other places in the system with this issue before merging.

File: .gdbinit
Comment (pholat) [reply .gdbinit:3]: 1. it's not needed
2. with it we ignore sigtrap in GDB by default which is terribly confusing

File: module-apps/apps-common/windows/OptionWindow.hpp
Comment (pholat) [reply module-apps/apps-common/windows/OptionWindow.hpp:39]: feel free to provide me a better name - it will be appreciated!  (there are better ways to fix this issue, none is that short and non-intrusive though)

**Review by PrzeBrudny — APPROVED** [review]

**Comment by pholat** [comment]
added rai class to handle all cases when we can call non existing class on destruction for the Settings windows

**Pull request merged**
